services/learn-card-network/brain-service/test/boosts.spec.ts

```diff
@@ -137,6 +137,45 @@ describe('Boosts', () => {
         });
     });
 
+    describe('getPaginatedBoosts', () => {
+        beforeEach(async () => {
+            await Profile.delete({ detach: true, where: {} });
+            await Credential.delete({ detach: true, where: {} });
+            await Boost.delete({ detach: true, where: {} });
+            await userA.clients.fullAuth.profile.createProfile({ profileId: 'usera' });
+            await userB.clients.fullAuth.profile.createProfile({ profileId: 'userb' });
+        });
+
+        afterAll(async () => {
+            await Profile.delete({ detach: true, where: {} });
+            await Credential.delete({ detach: true, where: {} });
+            await Boost.delete({ detach: true, where: {} });
+        });
+
+        it('should require full auth to get boosts', async () => {
+            await userA.clients.fullAuth.boost.createBoost({ credential: testVc });
+
+            await expect(noAuthClient.boost.getPaginatedBoosts()).rejects.toMatchObject({
+                code: 'UNAUTHORIZED',
+            });
+            await expect(
+                userA.clients.partialAuth.boost.getPaginatedBoosts()
+            ).rejects.toMatchObject({
+                code: 'UNAUTHORIZED',
+            });
+        });
+
+        it('should allow getting boosts', async () => {
+            await userA.clients.fullAuth.boost.createBoost({ credential: testVc });
+
+            await expect(userA.clients.fullAuth.boost.getPaginatedBoosts()).resolves.not.toThrow();
+
+            const boosts = await userA.clients.fullAuth.boost.getPaginatedBoosts();
+
+            expect(boosts.records).toHaveLength(1);
+        });
```

Comment:
Should we, for all the paginated queries, have a test for ensuring pagination is working ?

As in, testing 25 records will paginate 5 at a time or something, and will not skip records?

Reply:
I had thought about this, but I'm worried it will slow down an already very slow test suite =/ I can add them in if you want though! If you do, I'll try and think of the speediest way to get those guys in!